qwerty/programmer_english.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="Programmer (English)" script="latin">
+  <row>
+    <key c="q" nw="loc esc" se="(" ne="1"/>
+    <key c="w" se="{" ne="2"/>
+    <key c="e" se="!" ne="3"/>
+    <key c="r" sw="$" ne="4"/>
+    <key c="t" sw="%" ne="5"/>
+    <key c="y" sw="^" ne="loc redo" nw="6"/>
+    <key c="u" sw="&amp;" ne="7"/>
+    <key c="i" sw="*" ne="8"/>
+    <key c="o" sw="}" ne="9"/>
+    <key c="p" sw=")" ne="loc page_up" nw="0"/>
+  </row>
+  <row>
+    <key shift="0.5" se="[" c="a" sw="loc selectAll" nw="loc tab"/>
+    <key c="s" ne="_" se="-"/>
+    <key c="d" nw="\\"/>
+    <key c="f" nw="+"/>
+    <key c="g" nw="/"/>
+    <key c="h" ne="="/>
+    <key c="j" ne=";"/>
+    <key c="k" sw="&quot;" nw="'"/>
+    <key c="l" ne="|" sw="]" se="loc page_down"/>
+  </row>
+  <row>
+    <key width="1.5" c="shift" ne="loc capslock"/>
+    <key c="z" ne="&lt;" sw="loc undo"/>
```

Comment:
The undo, cut, copy, etc.. keys should already appear here when added in the settings when using the dev version of the app.

Reply:
Well, i am using the release version, so i had to map these manually.